azconfig/client/src/main/java/com/azure/azconfig/models/KeyFilters.java

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.azconfig.models;
+
+/**
+ * Key Filters.
+ */
+public class KeyFilters {
+    /**
+     * Matches any key.
+     */
+    public static final String ANY = "*";
+}
```

Comment:
This kind of class feels lame to me. I would be inclined to create a `public static final String KEY_FILTER_ANY = "*";` in the main AzConfigClient class instead.